src/RevPrefetcher.cc

```diff
@@ -58,6 +63,10 @@ bool RevPrefetcher::IsAvail(uint64_t Addr){
     }
   }
 
+  if( Addr == 0x4bd38ull ){
+    std::cout << "Filling 0x4bd38" << std::endl;
+  }
+
   // if we reach this point, then the instruction hasn't even triggered
```

Comment:
remove address check

Reply:
resolved in https://github.com/tactcomplabs/rev/commit/e9a5df3d3de81136677d389638c82e5ced6b65c9